ui/components/multichain/asset-list-conversion-button/asset-list-conversion-button.scss

```diff
@@ -0,0 +1,21 @@
+.asset-list-conversion-button {
+  min-width: 157px;
+  height: 56px;
+  flex-grow: 1;
+
+
+  &__contents {
+    flex-grow: 1;
+
+    &__button-wrapper {
+      background: var(--brand-colors-white-white000);
+      border-radius: 50%;
+      width: 24px;
+      height: 24px;
+    }
+
+    &__text {
+      color: var(--brand-colors-white-white000);
+    }
```

Comment:
Can we use the text color from Text Props

Reply:
Unfortunately not, as `--brand-colors-white-white000` isn't exposed as a named color (https://github.com/MetaMask/design-tokens/blob/main/src/css/design-tokens.css) . I'd prefer to use this var rather than hardcode `#fff`.  Also `--brand-colors-white-white000` is very very unlikely to ever change.  I can address some of this with @georgewrmarshall when he gets back, but this shouldn't block.